src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -147,6 +152,28 @@
  *   &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example to allow only one and only parameterized annotation on the same line
```

Comment:
Please add a colon `:` after this sentence

Reply:
Done